auth/src/main/java/com/firebase/ui/auth/ui/ChooseAccountActivity.java

```diff
@@ -219,14 +219,13 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
             case RC_IDP_SIGNIN:
             case RC_AUTH_METHOD_PICKER:
             case RC_EMAIL_FLOW:
-                finish(resultCode, new Intent());
+                finish(resultCode, data);
                 break;
             case RC_PLAY_SERVICES:
                 if (resultCode != RESULT_OK) {
-                    finish(resultCode, new Intent());
```

Comment:
Your change is correct, the `new Intent()` was in the absence of anything important to plumb down.